tests/DotNetLightning.Core.Tests/EncryptDecrypt.fs

```diff
@@ -0,0 +1,29 @@
+module EncryptDecryptTest
```

Comment:
Would you mind adding property test for bidirectionally converting random byte vector and asserting that it won't change?
i.e. `Expect.equal (randomBytes |> encryptWithAD |> decryptWithAd) randomBytes` (Some arguments omitted )

It should be not that hard.